scalableminds/webknossos
# Make organization name unique

Introducing a UNIQUE constraint to organization names, to avoid wrong lookups.

### Steps to test:
- change to a demo instance via application.conf (isDemoInstance=true)
- change L45 in spotlight_registration_form.js to a fixed string
- error message should show, should not be able to create multiple organizations with the same name
- creating organizations with different names should still be possible

### Issues:
- fixes #4675 

------
